tests/test_node_differ.py

```diff
@@ -198,4 +326,10 @@ def test_moved_section():
                                           1)
     expected_changes = {'Section':{'move':1, 'change':2}}
     diff = EditTypes(prev_wikitext, curr_wikitext, lang='en').get_diff()
+    assert expected_changes == diff
+
+def test_insert_cjk_punctuations():
+    cjk_curr_wikitext = cjk_prev_wikitext.replace('。','、',1)
+    expected_changes = {'Section':{'change':1}, 'Punctuation':{'change':1},'Sentence':{'change':1},'Paragraph':{'change':1}}
+    diff = EditTypes(cjk_prev_wikitext, cjk_curr_wikitext, lang='en').get_diff()
```

Comment:
not super relevant for this because it's not changing any categories/media but I'd set the `lang` parameter to `ja` for consistency/clarity.

Reply:
Fixed